### PR TITLE
Create `@spark-web/design-system` package

### PR DESCRIPTION
**Pull request by lukebennett88**
# Description

This PR creates a new package (`@spark-web/design-system`) that exports all the other design system packages.

The package seems to work, but for some reason, it breaks Playroom.

### CODE REVIEW

**Comment by lukebennett88** [comment]
For some reason Playroom didn't like it when I did this 🤔

File: scripts/generate-design-system-package.mjs [hunk 1]
```diff
@@ -39,9 +39,8 @@ const packageNames = readdirSync(PACKAGE_PATH, { withFileTypes: true })
...
 
 mkdirSync(MANIFEST_DIR, { recursive: true });
+
 writeFileSync(
-  `${MANIFEST_DIR}/spark-components.ts`,
-  ['../components/example-helpers', ...packageNames]
-    .map(name => `export * from '${name}';`)
-    .join('\n')
+  `${MANIFEST_DIR}/index.ts`,
+  packageNames.map(name => `export * from '${name}';`).join('\n')
```
Comment (lukebennett88) [inline scripts/generate-design-system-package.mjs:45]: Doesn't account for packages with multiple entrypoints (at this stage it's just `@spark-web/utils`.

File: docs/package.json [hunk 2]
```diff
@@ -6,58 +6,21 @@
   "scripts": {
     "build": "npm-run-all --sequential build:*",
     "build:contentlayer": "INIT_CWD=$PWD yarn contentlayer build",
-    "build:docs": "yarn generate-components-index && INIT_CWD=$PWD next build",
+    "build:docs": "INIT_CWD=$PWD next build",
```
Comment (lukebennett88) [inline docs/package.json:9]: Removing the `generate-components-index` script as all the components are in `@spark-web/design-system`

**Review by nderkim — APPROVED** [review]

**Pull request merged**